src/pages/[domain]/admin/hirer-profile.tsx

```diff
@@ -1,13 +1,24 @@
 import { getBuilderPlace } from '../../../modules/BuilderPlace/queries';
+import ProfileForm from '../../../components/Form/ProfileForm';
+import BuilderPlaceContext from '../../../modules/BuilderPlace/context/BuilderPlaceContext';
+import { useContext } from 'react';
+import AccessDenied from '../../../components/AccessDenied';
 
 export async function getServerSideProps({ params }: any) {
   return await getBuilderPlace(params.domain);
 }
 
 export default function HirerProfile() {
+  const context = useContext(BuilderPlaceContext);
+
+  if (!context.isBuilderPlaceOwner) {
+    return <AccessDenied />;
+  }
+
   return (
-    <div>
-      <h1>hirer profile</h1>
-    </div>
+    <>
+      <h1>Hirer profile</h1>
+      <ProfileForm context={context} />
```

Comment:
context is a lot of thing, not really clean to do that, need to pass precisely the user to update.

Reply:
it will prevent to have to manage a double typing, the code will be more robust, lisible and predictable